lib/tasks/remove_repositories_without_contributions.rake

```diff
@@ -0,0 +1,6 @@
+namespace :remove_repositories do
+  desc 'Remove repositories without contributions'
+  task without_contributions: :environment do
```

Comment:
What do u think about renaming this task to `clean_up_repositories_with_no_contribs`?